### PR TITLE
SceNpTrophy: partial implementation of sceNpTrophyGetGameInfo and sceNpTrophyGetTrophyInfo

### PR DESCRIPTION
**Pull request by scribam**

### CODE REVIEW

**Comment by Zangetsu38** [comment]
you can complete your codee with using this
https://github.com/Zangetsu38/Vita3K/commit/48f4655cfd7958ee4e8cc047f048b8145d926076#diff-75b1da0b7345555b2e1a2b52879015d2189f7d8ac9a1cbcc37ad2b4a04a11d23R105
i have old time start implement this function

**Comment by scribam** [comment]
@Zangetsu38 Feel free to do it yourself 😉

**Comment by Zangetsu38** [comment]
hmm i see, i have thinks you are better me for more proper code lol

**Comment by Zangetsu38** [comment]
ah yers sorry, you can move struct and define in include like here ? 
https://github.com/Zangetsu38/Vita3K/commit/48f4655cfd7958ee4e8cc047f048b8145d926076#diff-7a3b0c65a2e18a15badb9f01cdf350cdeb47cb8a8217a176e3125e42dcda80acR26

File: vita3k/modules/SceNpTrophy/SceNpTrophy.cpp [hunk 1]
```diff
@@ -21,6 +21,51 @@
 
 #include "SceNpTrophy.h"
 
+#define SCE_NP_TROPHY_GAME_TITLE_MAX_SIZE 128
```
Comment (Zangetsu38) [inline vita3k/modules/SceNpTrophy/SceNpTrophy.cpp:24]: move this def and struct inside SceNpTrophy.h like other already present

**Review by Zangetsu38 — CHANGES_REQUESTED** [review]

**Review by Zangetsu38 — APPROVED** [review]

**Pull request merged**